.gitignore

```
@@ -6,7 +6,7 @@
 /out-tsc
 
 # dependencies
-/node_modules
+node_modules
```

Please revert this change

```suggestion
/node_modules
```